src/Domain/Fixture/FixtureCreator.php

```diff
@@ -257,6 +257,9 @@ private function fixDefaultDependencies(): void {
       $additions[] = 'phpunit/phpunit:9.4.3';
     }
 
+    // Hack to support Acquia CMS.
```

Comment:
We should have a `@todo` comment in here so we don't lose track of it. If we create a follow-up issue, we can add/replace it with an `@see`.
```diff
-    // Hack to support Acquia CMS.
+    // Hack to support Acquia CMS. Any more detail?
+    // @todo Remove once... ???
```

Reply:
Added a todo.